tests/cnf/core/network/internal/netconfig/config.go

```diff
@@ -38,6 +38,7 @@ type NetworkConfig struct {
 	//nolint:lll
 	PrometheusOperatorNamespace string `yaml:"prometheus_operator_namespace" envconfig:"ECO_CNF_CORE_NET_PROMETHEUS_OPERATOR_NAMESPACE"`
 	MlbAddressPoolIP            string `envconfig:"ECO_CNF_CORE_NET_MLB_ADDR_LIST"`
+	SecurityIPAddList           string `envconfig:"ECO_CNF_CORE_NET_SECURITY_ADDR_LIST"`
```

Comment:
Why do we need it? Is MlbAddressPoolIP not good for that?

Reply:
We can use it. Its the same addresses

Reply:
removed